File: Modulo_3/Dp420Conexao/DTO/ProductDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Dp420Conexao.DTO
{
    public class ProductDTO
    {
        
        public string? CategoriaId  {get;set;}
        public string? Name {get;set;}
        public double Price {get;set;}
        public string[]? Tags {get;set;}
    }
}

File: Modulo_3/Dp420ConexaoJava/TesteCosmoDB/src/main/java/com/nosql/TesteCosmoDB/DTO/ProductDTO.java
package com.nosql.TesteCosmoDB.DTO;

import java.math.BigDecimal;
import java.util.UUID;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProductDTO {
    
    private String categoryId;
    private String name;
    private BigDecimal price;
    private String[]  tags; 
}
